src/App.jsx

```diff
@@ -1,13 +1,96 @@
-
-import './App.css'
+import './App.css';
+import './css-reset.css';
+import FruitCounter from './components/fruit-counter/fruit-counter.jsx';
+import React, { useState } from 'react';
+import OrderForm from './components/order-form/order-form.jsx';
 
 function App() {
+  const initialProductState = [
+    { name: 'Aardbeien', emoji: '🍓', counter: 0, id: 0 },
+    { name: 'Bananen', emoji: '🍌', counter: 0, id: 1 },
+    { name: 'Appels', emoji: '🍏', counter: 0, id: 2 },
+    { name: 'Kiwi\'s', emoji: '🥝', counter: 0, id: 3 },
+  ];
+  const initialFormState = {
+    voornaam: '',
+    achternaam: '',
+    leeftijd: '',
+    postcode: '',
+    bezorgfrequentie: '',
+    bezorgtijd: '',
+    opmerking: '',
+    voorwaarden: false,
+  };
+
+  const [products, setProducts] = useState(initialProductState);
+  const [form, setForm] = React.useState(initialFormState);
+
+  function incrementCounter(id) {
```

Comment:
Keurige increment en decrement functies. Het zou ook mogelijk zijn geweest om maar 1 functie te maken waarin een ternary operator zit die kijkt naar de parameters. Vervolgens is het afhankelijk van die parameters wordt het increment of decrement.